aml-development/frontend
# feat(theme): patriotic and forest theme

AMLOS-815

**Description**
Look at https://wiki.udev.bia-boeing.com/display/AML/Themes and implement the patriotic and forest themes as per the style guides.

**Acceptance Criteria**
- Patriotic Theme is available and matches the style guides
- Forest Theme is available and matches the style guides

**How To Test**
Pull this branch
Select the theme options from the hamburger menu (if run in dev mode) or by editing profile
Run through and make sure site still looks good
